common/datasets/librispeech.py

```diff
@@ -415,16 +415,19 @@ def get_g2p_augmented_bliss_lexicon_dict(use_stress_marker=False, subdir_prefix=
     bliss_corpus_dict = get_bliss_corpus_dict(subdir_prefix=subdir_prefix)
     for corpus_name, bliss_corpus in sorted(bliss_corpus_dict.items()):
         if "train" in corpus_name:
-            g2p_augmenter = G2PBasedOovAugmenter(
-                original_bliss_lexicon=updated_bliss_lexicon,
-                train_lexicon=original_bliss_lexicon,
-            )
-            current_bliss_lexicon = g2p_augmenter.get_g2p_augmented_bliss_lexicon(
-                bliss_corpus=bliss_corpus,
-                corpus_name=corpus_name,
-                alias_path=alias_path,
-            )
-            augmented_bliss_lexica[corpus_name] = current_bliss_lexicon
+            if corpus_name in ["train-clean-460", "train-other-960"]:
```

Comment:
I might be asking too much, but wouldn't the cleaner solution here be: run the OOV job, compare the resulting OOV set with the set of the previous step, and then decide?

Reply:
Sisyphus would support this kind of "Graph decision" only with the async workflows, and we do not want to introduce this here.

Reply:
I see. My only concern is that this is based on prior knowledge. Then I suggest that we document it at least.

Reply:
Ok then I will accept this PR, and add documentation, do some testing, and if everything is fine add also the new lexicon stuff to /work/common